app/src/main/java/com/example/hamlet/api/Retrofit.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.hamlet.api
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class Retrofit {
+    val retrofit = Retrofit.Builder()
```

Comment:
Rename this class to something else and not retrofit to avoid conflict with the Retrofit class from Square. You can call it RetrofitProvider.

Also put the Retrofit creation in an init {} block.

Reply:
Alright, boss I'll do that. Thanks